Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -388,7 +388,8 @@ struct atom_wrapper {
 
         .def("GetPropsAsDict", GetPropsAsDict<Atom>,
              (python::arg("self"), python::arg("includePrivate") = true,
-              python::arg("includeComputed") = true),
+              python::arg("includeComputed") = true,
+	      python::arg("autoConvert") = true),
```

Comment:
I think the name of this argument should be `autoConvertStrings` to make it more explicit when the autoconversion actually happens.